openapi.yaml

```diff
@@ -941,6 +941,10 @@ definitions:
                 description: The program classification of the major
                 type: string
                 example: Spanish Language & Literature
+          honorInd:
+            description: Determine if student is honor for this degree
```

Comment:
I think this should be `Determines if this degree is an honors degree`

Reply:
Sounds good, fixed.